library/ssl_tls13_generic.c

```diff
@@ -4358,6 +4486,13 @@ int mbedtls_ssl_conf_ticket_meta( mbedtls_ssl_config *conf,
 #endif /* MBEDTLS_SSL_NEW_SESSION_TICKET */
 
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) && defined(MBEDTLS_X509_CRT_PARSE_C)
```

Comment:
Do we still need to check for `MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL` in `library/ssl_tls13_generic.c `?

Reply:
I actually don't need it.